.circleci/config.yml

```diff
@@ -0,0 +1,71 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:10.15.1
+    steps:
+      - checkout
+      - run:
+          command: npm ci
+      - run:
+          command: npm run build
+      - persist_to_workspace:
+          root: .
+          paths:
+            - build
+
+  deploy:
+    docker:
+      - image: olizilla/ipfs-dns-deploy
+        environment:
+          DOMAIN: explore.ipld.io
+          BUILD_DIR: build
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - run:
+          name: Pin website, post notification for PRs or update DNS on master
+          command: |
+            # Upload build dir to cluster.ipfs.io
+            hash=$(ipfs-cluster-ctl \
+              --host /dnsaddr/cluster.ipfs.io \
+              --basic-auth $CLUSTER_USER:$CLUSTER_PASSWORD \
+              add --rmin 3 --rmax 3 --name "$DOMAIN" \
+              --recursive /tmp/workspace/$BUILD_DIR | tail -n1 | cut -d " " -f 2 )
+
+            preview_url=https://ipfs.io/ipfs/$hash
+
+            # Preload ipfs hash on the gateway
+            curl --silent --output /dev/null --show-error $preview_url
```

Comment:
Was the intention to preload just the HTML of main page?
If not, entire thing could be preloaded asynchronously in best-effort fashion without breaking the build:

```bash
nohup curl --max-time 900 -s "https://ipfs.io/api/v0/refs?r=true&arg=${$preview_url}" >/dev/null 2>&1 &
```